tools/sigclient/functionalQueries/functionalQueries.yml

```diff
@@ -1,23 +1,23 @@
-3stats.json
-9stats.json
-complex_search.json
-eval_groupby_2col_2stats.json
-grpby_1col_4stats.json
-search_1col_fields.json
-rare.json
-sort_tail_eval_newField.json
-stats_groupby_with_eval.json
-streamstats_win3.json
-streamstats_win4_currF.json
-streamstats_win4_grpby.json
-streamstats_win4_reset_before_after.json
-streamstats_win2_gloF_currF_grpby.json
-streamstats_win2_gloF_grpby.json
-streamstats_multi_resets_grpby.json
-streamstats_eval.json
-streamstats_after_stats.json
-streamstats_after_grpby.json
-multi_value.json
+# 3stats.json
+# 9stats.json
+# complex_search.json
+# eval_groupby_2col_2stats.json
+# grpby_1col_4stats.json
+# search_1col_fields.json
+# rare.json
+# sort_tail_eval_newField.json
+# stats_groupby_with_eval.json
+# streamstats_win3.json
+# streamstats_win4_currF.json
+# streamstats_win4_grpby.json
+# streamstats_win4_reset_before_after.json
+# streamstats_win2_gloF_currF_grpby.json
+# streamstats_win2_gloF_grpby.json
+# streamstats_multi_resets_grpby.json
+# streamstats_eval.json
+# streamstats_after_stats.json
+# streamstats_after_grpby.json
+# multi_value.json
```

Comment:
Why have we commented these out, are these not working?

Reply:
No they are working. I forgot to uncomment them.